### PR TITLE
options/bsd: Add fstab.h header

### PR DESCRIPTION
**Pull request by Dennisbonke**
Part of the mlibc LFS project.

### CODE REVIEW

File: options/bsd/include/fstab.h [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+	char *fs_spec;
+	char *fs_file;
+	char *fs_vfstype;
+	char *fs_mntops;
+	const char *fs_type;
+	int fs_freq;
+	int fs_passno;
+};
+
+#endif // _FSTAB_H
```
Comment (ArsenArsen) [inline options/bsd/include/fstab.h:23]: Don't add C++-style comments in public interfaces.

**Review by ArsenArsen — CHANGES_REQUESTED** [review]
What programs include this header in order to not use getfsent and friends?

**Comment by Dennisbonke** [comment]
> What programs include this header in order to not use getfsent and friends?

`libsanitizers` makes use of this header.

**Comment by ArsenArsen** [comment]
Looks like it's only used to determine the size of this header (couldn't the gcc devs come up with a better way to enumerate the systems structs?)

**Review by Geertiebear — APPROVED** [review]
LGTM.

**Pull request merged**